data/json/mapgen/nested/lmoe_nested.json

```diff
@@ -84,5 +84,15 @@
         }
       ]
     }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "This is done to give a chance for lmoe bunkers to spawn with solar panels on the roof",
+    "nested_mapgen_id": "lmoe_solar",
+    "object": { "mapgensize": [ 2, 2 ], "rows": [
```

Comment:
You'll need to define `M` here though

Reply:
@Procyonae You've been working with nested mapgen a lot lately.  Nests can't use palettes right? Or am I remembering incorrectly?

Reply:
Nests can use palettes just fine